ivy/functional/frontends/numpy/__init__.py

```diff
@@ -678,4 +682,49 @@ def promote_types_of_numpy_inputs(
 divmod = ufunc("_divmod")
 fmax = ufunc("_fmax")
 ldexp = ufunc("_ldexp")
+
+
+
+
+# flake8: noqa
+from . import creation_routines
```

Comment:
I think this is old changes from old commit, please also remove these (repetition with the existing code as well)